Week4 treat bug

ishaanv opened the pull request:
Attain github username from remote url

Commit:
:bug: get github username from git config

Commit:
:wrench: check if ssh url, call fn with updated args

notionparallax commented on week4/tests.py:
```diff
@@ -47,11 +47,22 @@ def find_lasers(path):
+
+    with open('.git/config', 'r') as f:
+        for line in f:
+            if 'code1161base' in line and not 'notionparallax' in line:
```
The problem with this is that not all the repos are called code1161base.
I like the theory though.

notionparallax committed:
More exercises

notionparallax committed:
Don't bump out the first line

notionparallax committed:
check for something more guaranteed

I think this is safer than checking for code1161base as some have renamed their repo, although it may trip on ssh

notionparallax committed:
make tz locale specific

notionparallax committed:
Make test give some feedback

notionparallax committed:
Make testing specific condition

notionparallax committed:
Bullshit commit

notionparallax committed:
Bullshit commit

notionparallax committed:
4-treat-bug: Auto stash for revert of "Revert "sacrificial""

notionparallax committed:
comment out unused import

notionparallax committed:
get rid of all the stars

notionparallax committed:
cap timeout at 25

notionparallax committed:
Add timeout controls

notionparallax committed:
Clearer coment

notionparallax commented:
This was such a ballache to merge! Because I'd been working on the file too there were bits all over the place!

Could you test the updated version please?
